const.go

```diff
@@ -32,6 +32,8 @@ const (
 
 	CMD_NEXT             = "next"
 	CMD_ADD              = "add"
+	CMD_RM               = "rm"
```

Comment:
I'd rather not have aliases baked in, as the tab completion for dstask makes it a bit redundant. Though, `rm `is muscle memory for most people that use a shell; perhaps this command should be `rm ` only

Reply:
Actually this is apparently fine as I've already done this for other commands. I guess I'd rather be consistent, let's leave remove/rm in.